src/main/kotlin/no/nav/familie/tilbake/behandlingskontroll/domain/Behandlingsstegstilstand.kt

```diff
@@ -21,18 +22,20 @@ data class Behandlingsstegstilstand(@Id
                                     @Embedded(onEmpty = Embedded.OnEmpty.USE_EMPTY)
                                     val sporbar: Sporbar = Sporbar())
 
-enum class Behandlingssteg(val sekvens: Int, val kanSaksbehandles: Boolean, private val beskrivelse: String) {
-
-    VARSEL(1, false, "Vurdere om varsel om tilbakekreving skal sendes til søker"),
-    GRUNNLAG(2, false, "Mottat kravgrunnlag fra økonomi for tilbakekrevingsrevurdering"),
-    VERGE(3, true, "Fakta om verge"),
-    FAKTA(4, true, "Fakta om Feilutbetaling"),
-    FORELDELSE(5, true, "Vurder om feilutbetalte perioder er foreldet"),
-    VILKÅRSVURDERING(6, true, "Vurdere om og hva som skal tilbakekreves"),
-    FORESLÅ_VEDTAK(7, true, "Foreslår vedtak"),
-    FATTE_VEDTAK(8, true, "Fatter vedtak"),
-    IVERKSETT_VEDTAK(9, false, "Iverksett vedtak fra en behandling.  Forutsetter at et vedtak er fattet"),
-    AVSLUTTET(10, false, "Behandlingen er ferdig behandlet");
+enum class Behandlingssteg(val sekvens: Int, val kanSaksbehandles: Boolean,
+                           val behandlingsstatus: Behandlingsstatus, private val beskrivelse: String) {
```

Comment:
Code style sier at chop-down skal brukes på parametere hvis linen blir for lang. Dvs. hvis du bruker linkeskift for å skille parameterne, så skal det bare være en parameter per linje. Dette kommer av seg selv hvis du bruker Ctrl+Alt+L i setdet for å legge inn linjeskift manuelt.